Education.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;

```

Comment:
Please remove references that you are not using

Reply:
Yeah sure. I'll take care of it on all the classes